.circleci/config.yml

```diff
@@ -2,20 +2,42 @@ version: 2.1
 
 orbs:
   ft-golang-ci: financial-times/golang-ci@1
+  docker: circleci/docker@1.7.0
+
+jobs:
+  build-and-publish-docker-image:
+          docker:
+            - image: docker:stable-git
+          steps:
+            - checkout
+            - setup_remote_docker
+            - run:
+                name: Build Docker image
+                command: docker build --tag coco/publish-failure-resolver-go:latest .
+            - run:
+                name: Login in Dockerhub
+                command: echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_LOGIN" --password-stdin
```

Comment:
While this seems correct, I'm wondering whether it's better to use the `dockerhub-shared` context instead of including both of these variables on the project level in CircleCI.